packages/mdc-typography/addon/instance-initializers/typography.js

```diff
@@ -16,7 +16,7 @@ export function initialize (app) {
   const { disabled = false} = config;
 
   if (!disabled) {
-    const rootElement = document.querySelector (app.rootElement);
+    const rootElement = typeOf(app.rootElement) === 'string' ? document.querySelector (app.rootElement) : app.rootElement;
```

Comment:
@mtsgeneroso I am working on coding guidelines for this project. In the meantime, can you add a space after `typeOf` and before the parenthesis.

    typeOf (app.rootElement)

Reply:
Nice, it will be very helpful.